dotcom-rendering/src/web/lib/useApi.tsx

```diff
@@ -40,6 +40,6 @@ export const useApi = <T,>(
 	return {
 		data,
 		error,
-		loading: !error && !data,
+		loading: url !== undefined && !error && !data,
```

Comment:
```suggestion
		loading: url && !error && !data,
```

Reply:
I considered this but I this would return false if `url` was an empty string, right? But I thought maybe we don't want to encourage passing an empty string as a way of preventing an API call from being made here, so it would be good to restrict it to `undefined` (or `null` if we're willing to refactor later).

On the other hand, if someone passes an empty string by mistake, your suggestion would be more forgiving at runtime, I think, so might be preferable.

I don't have strong opinions either way so lmk what you think and I'm happy to change it if so

Reply:
I think the SWR docs assume that if falsey, no request will be made, so I'd tend to follow that same pattern here!

Reply:
sounds good, change accepted -- thanks!